sympy/combinatorics/perm_groups.py

```diff
@@ -3018,6 +3027,19 @@ def strong_gens(self):
             self.schreier_sims()
         return self._strong_gens
 
+    def subgroup(self, gens):
+        """
+           Return the subgroup generated by `gens` which is a list of
+           elements of the group
+        """
+
+        if not all([g in self for g in gens]):
+            raise ValueError("The group doesn't contain the supplied generators")
+
+        G = PermutationGroup(gens)
+        G._parent = self
```

Comment:
I suppose it's not as relevant to permutation groups. For `FpGroup`s it allows to have `is_subgroup` method, and also this is a set-up GAP has (i.e. having `.parent` attribute for subgroups of groups). Theoretically having this might save time when someone tries to call `is_subgroup` on large permutation groups but mainly I was just mimicking what I did for `FpGroup`s here. Should I remove it?